docs/privacy-policy.rst

```diff
@@ -0,0 +1,361 @@
+and this may include "User Personal Information."
+
+"User Personal Information" is any information about one of our users which could,
+alone or together with other information, personally identify him or her.
+Information such as a user name and password, an email address,
+a real name, and a photograph are examples of "User Personal Information."
+
+User Personal Information does not include aggregated, non-personally identifying information.
+We may use aggregated, non-personally identifying information to operate, improve,
+and optimize our website and service.
+
+Why do we collect this?
++++++++++++++++++++++++
+
+- We need your User Personal Information to create your account, and to provide the services you request.
+- We use your User Personal Information, specifically your user name, to identify you on Read the Docs.
+- We use it to fill out your profile and share that profile with other users.
+- We will use your email address to communicate with you but it is not shared publicly.
+- We limit our use of your User Personal Information to the purposes listed in this Privacy Statement.
+  If we need to use your User Personal Information for other purposes, we will ask your permission first.
+  You can always see what information we have in your
+  `user account <https://readthedocs.org/accounts/edit/>`_.
+
+What information Read the Docs does not collect
+-----------------------------------------------
+
+We do not intentionally collect **sensitive personal information**,
+such as social security numbers, genetic data, health information, or religious information.
+
+Documentation Sites hosted on Read the Docs are public,
+anyone (including us) may view their contents.
+If you have included private or sensitive information in your Documentation Site,
+such as email addresses, that information may be indexed by search engines or used by third parties.
+
+If you're a **child under the age of 13**, you may not have an account on Read the Docs.
+Read the Docs does not knowingly collect information from or direct any of our content specifically to children under 13.
+If we learn or have reason to suspect that you are a user who is under the age of 13, we will unfortunately have to close your account.
+We don't want to discourage you from writing software documentation, but those are the rules.
+
+How we share the information we collect
+---------------------------------------
+
+We **do not** share, sell, rent, or trade User Personal Information with
+third parties for their commercial purposes.
+
+We do not disclose User Personal Information outside Read the Docs,
+except in the situations listed in this section or
+in the section below on compelled disclosure.
+
+We **do** share certain aggregated, non-personally identifying information
+with others about how our users, collectively, use Read the Docs.
+For example, we may compile statistics on the prevalence of
+different types of documentation across Read the Docs.
```

Comment:
My question as a user here would be "who does this go to?"

"others" could be more specific -- "advertising partners" or whatever.